Big ui revision.

gemini6 opened the pull request:
Hey StevesBrain. If you read the IMPORTANT_NOTE.txt file it explains why I did this but feel free to use this or not. Its best viewed so far on small devices such as phones and tablets (under 768px) I made it semi-responsive so far. I love the header and footer on a phone or tablet but on larger screens I don't know what I want to do yet. It just doesn't look as good anymore to me.

gemini6 committed:
Added project files.

Added the revisions to the start page that I made to the project directory and removed the original version.

gemini6 committed:
Files added

I returned the startpage.html and style.css files to the project directory.

gemini6 committed:
Added file

I added the IMPORTANT-NOTE.txt file to the project.

gemini6 committed:
UI update.

Added a blur effect to all of the main-nav-links of the currently hovered over main-nav-row that are not being hovered over.

gemini6 committed:
Added hey tia

gemini6 committed:
Style revision.

Changed the color of the blur effect of the .main-nav-row links. I might revise it again in the future.

gemini6 committed:
UI Update & project structure revision.

- Created the css folder and moved all css files into it.
- Created the js folder.
- Removed the internal ui scripts and put them in ui-animations.js which I created and moved into the js folder.
- created the libs folder.
- Included animate.css to the project for the page load slide in effect on the main-nav-rows.
- Included a local copy of animate .css to the libs folder.
- created responsive.css and started writing the media queries for larger devices.

gemini6 commented on index.html:
```diff
@@ -1,3 +1,4 @@
+hey tia
```
I was showing my gf how commits work on GitHub. :P

gemini6 committed:
Update IMPORTANT-NOTE.txt

gemini6 committed:
Switched back to original bg color.

#2D2D2D

gemini6 committed:
Page revision.

Revisions to both index.html and index.css.

stevesbrain commented:
Thanks @gemini6 - will check these changes out in the next day or two; just on holidays at the moment! Thanks  in advance! :)

gemini6 commented:
No worries. Enjoy.

On Jan 10, 2018 3:10 AM, "Steve Divskinsy" <notifications@github.com> wrote:

> Thanks @gemini6 <https://github.com/gemini6> - will check these changes
> out in the next day or two; just on holidays at the moment! Thanks in
> advance! :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/stevesbrain/html/pull/2#issuecomment-356529904>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AeVM21147LKkC1wCiSLPfOhACyRSQPfWks5tJHBtgaJpZM4RV45s>
> .
>

gemini6 committed:
Updated IMPORTANT-NOTE.txt

gemini6 committed:
Project revision.

- Cleaned up the css a little bit.
- Added the .header-link & .header-link-icon classes.

gemini6 commented:
@gemini6

Note: The sight is fully responsive now and viewable on any screen size. Cheers.

gemini6 reviewed (commented):
The local copy of animate.css is not required as the cdn is available and being used. I just threw it in in case anyone wanted to use it locally instead.

gemini6 committed:
Small revision:

- Removed the background color from the .navbar-nav on small screens.

-Added notes-on-index-html.txt file

stevesbrain commented:
All merged - thanks @gemini6 :)
